src/Host/Broker/Impl/Sessions/SessionsController.cs

```diff
@@ -54,15 +56,20 @@ public class SessionsController : Controller {
                 interp = _interpManager.Interpreters.First();
             }
 
-            var session = _sessionManager.CreateSession(User.Identity, id, interp, securePassword, profilePath, request.CommandLineArguments);
-            return Task.FromResult<IActionResult>(new ObjectResult(session.Info));
+            try {
+                var session = _sessionManager.CreateSession(User.Identity, id, interp, securePassword, profilePath, request.CommandLineArguments);
+                return Task.FromResult<IActionResult>(new ObjectResult(session.Info));
+            } catch (Exception ex) {
```

Comment:
Limit to Win32Exceptions? User may still want more information for bug reports - logging may be disabled in some cases.